Automattic/wp-calypso
# Use Preact in standalone notifications

Standalone notifications are meant to be a widget loaded on every page view for logged-in users, and as such are critical to optimise.

Preact is a much better fit for this scenario than React, as it's much smaller, while essentially being a drop-in replacement.

This PR replaces React with Preact for the notifications build, through build-time aliasing in webpack and Babel. It should result in savings of around ~130KB, or ~40KB gzipped, for the notifications bundle.

#### Changes proposed in this Pull Request

* Replace React with Preact at build-time for the standalone notifications app

#### Testing instructions

Please follow the standard testing instructions for standalone notifications, and smoke-test them to ensure nothing breaks. Matticbot will likely add the instructions below, but in case it doesn't, here they are:

> To test your changes on WordPress.com, run `install-plugin.sh notifications update/lodash-es-on-notifications` on your sandbox.
> 
> To deploy your changes after merging, see the documentation: PCYsg-elI-p2
